rescript/get_unite.py

```diff
@@ -0,0 +1,185 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2021, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import os
+import tempfile
+import tarfile
+import requests
+from requests.exceptions import HTTPError
+
+import qiime2
+
+
+def _unite_get_doi(version, taxon_group, singletons):
+    """Lookup UNITE DOIs from included list"""
+    # Lookup DOIs for databases, see: https://unite.ut.ee/repository.php
+    unite_dois = {
```

Comment:
Set this dictionary as a global variable. This will make it easier to update and use for both other code and especially within unit tests. For example, see [here](https://github.com/bokulich-lab/RESCRIPt/blob/946728467532f066c8bb750c3d8d1b8af02773b9/rescript/parse_silva_taxonomy.py#L23) and [here](https://github.com/bokulich-lab/RESCRIPt/blob/946728467532f066c8bb750c3d8d1b8af02773b9/rescript/get_gtdb.py#L24).